modules/tensor_mechanics/tests/linear_elastic/tests

```diff
@@ -1,10 +1,4 @@
 [Tests]
-  [./tween]
-    type = 'Exodiff'
-    input = 'tween.i'
```

Comment:
I'd rather not. I'm referring to this file in the deprecation documentation

http://mooseframework.org/wiki/PhysicsModules/TensorMechanics/Deprecations/LinearElasticMaterial/